draft-ietf-quic-invariants.md

```diff
@@ -334,8 +334,8 @@ requires that middleboxes retain state for every connection ID they see.
 
 The Version Negotiation packet described in this document is not
 integrity-protected; it only has modest protection against insertion by off-path
-attackers.  QUIC versions MUST define a mechanism that authenticates the values
-it contains.
+attackers.  A QUIC version that uses a Version Negotiation packet MUST define a
```

Comment:
One could still argue that QUIC v1 "uses" the VN packet to indicate lack of support.

Reply:
```suggestion
attackers.  A specification which permits client to change the QUIC version in use
based on the contents of a Version Negotiation packet MUST also define a
```
Something like this?